packages/CodingStyle/tests/Rector/Assign/UseIncrementAssignRector/Fixture/increment-fixture.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\Assign\UseIncrementAssignRector\Fixture;
+
+class SomeClass
```

Comment:
The class name must be unique per namespace. Anything will fail in the future eventualy.

You can add all these changes to one file/method, since it's very simple change

Reply:
I hope i understood you, i've merged the fixtures for increment and decrement in one fixture file

Reply:
Yes, exactly :+1: